target/linux/ath79/dts/ar9341_openmesh_om2p-hs.dtsi

```diff
@@ -0,0 +1,163 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar9341.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	chosen {
+		/delete-property/ bootargs;
+	};
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &led_power_blue;
+		led-failsafe = &led_power_blue;
+		led-running = &led_power_blue;
+		led-upgrade = &led_power_blue;
+		label-mac-device = &eth1;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&jtag_disable_pins &led_lan_wan_blue_pin>;
```

Comment:
What about
`ath79_gpio_output_select(OM2P_GPIO_BTN_RESET, AR934X_GPIO_OUT_GPIO);`
?

Reply:
This line doesn't make a lot of sense. ath79_gpio_output_select switches the GPIO_OUT_FUNCTION registers for the output of an GPIO pin. But the OM2P_GPIO_BTN_RESET is an input. To switch the pin 1 to GPIO *input*, you would have to change 0x18040044- 0x1804006C (make sure that nothing uses the pin 1 for something else than GPIO). The relevant change to enable the input functionality for this button was actually the jtag_disable_pins because the pinmux has the pin 1 by default configured as JTAG TDI.